doc/user-how-to/troubleshooting.md

```diff
@@ -1,6 +1,62 @@
 # Troubleshooting
 
-## "Too many open files" on Linux
+```{contents} Contents
+:local:
```

Comment:
`:local:` prevented the top-level "Troubleshooting" header from being included in the output.